runtime/lua/vim/lsp/log.lua

```diff
@@ -8,7 +8,7 @@ local log = {}
 -- Log level dictionary with reverse lookup as well.
 --
 -- Can be used to lookup the number from the name or the name from the number.
--- Levels by name: 'trace', 'debug', 'info', 'warn', 'error'
+-- Levels by name: 'TRACE', 'DEBUG', 'INFO', 'WARN', 'ERROR'
 -- Level numbers begin with 'trace' at 0
```

Comment:
```suggestion
-- Level numbers begin with "TRACE" at 0
```

Reply:
Oh, this doesn't have to be double quotes as it is currently ignored by `gen_vimdoc.py`.

Reply:
But maybe it is better to make the comments in the three places consistent.